x-pack/plugins/fleet/public/components/uninstall_command_flyout/hooks.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+
+import { REDUCED_PLATFORM_OPTIONS } from '../../hooks';
+
+import type { Commands, UninstallCommandTarget } from './types';
+
+// todo: update with real API
```

Comment:
Do we have a follow up issue for the real API?

Reply:
yes, here it is: https://github.com/elastic/security-team/issues/6004